src/Behavioral.Automation/FluentAssertions/Assert.cs

```diff
@@ -68,12 +68,12 @@ public static void ShouldBecome<T>(Func<T> predicate, T value, string message, b
             True(actual.Equals(value) == direction, message);
         }
 
-        public static void ShouldBe(IAssertionAccessor assertion, string caption)
+        public static void ShouldBe(IAssertionAccessor assertion, string caption, int attempts = DefaultAttempts)
         {
             bool isValid = WaitForAssertion(assertion, TimeSpan.FromMilliseconds(500));
             if (assertion.Type == AssertionType.Continuous && (!isValid || !assertion.InterruptValidationOnSuccess))
             {
-                for (int i = 0; i < Attempts - 1; i++)
+                for (int i = 0; i < attempts - 1; i++)
                 {
                     Thread.Sleep(500);
```

Comment:
Looks like we do not need this command here.

Reply:
It may brake some tests. I propose to create a refactoring ticket if we feel a need to remove it

Reply:
Accepted.